Makefile

```diff
@@ -105,3 +105,13 @@ fmt:
 	find . '(' -name '*.h' -o -name '*.c' ! -path './vendor/*' ')'  -exec clang-format -i {} \+
 	find . -name '*.go' ! -path './vendor/*' -exec gofmt -s -w {} \+
 	git diff --exit-code
+
+
+.PHONY: dbuild
+dbuild:
+	-mkdir -p bin
+	-podman rm conmon-devenv
```

Comment:
Is the starting dash (`-`) correct in this line and the one prior?

Reply:
It depends.

With the `mkdir` the `-p` might be enough. I'll remove it if you consider it better. But I also don't see how it could hurt.

With the `podman rm` I think it's good because:

```
14:57:54:conmon🍔  podman rm i-do-not-exist
Error: no container with name or ID i-do-not-exist found: no such container
14:58:00:conmon🍔  echo $?
1                                                       <- it would error
```

Reply:
@rhdojun I'm fine with or without it.  I was just questioning as  I don't recall seeing one before and wasn't sure if it was meant to be there, and if so, what it's purpose was.

Reply:
Ah I see. Usually make exits with an error code when a command fails. With the dash before the command you can ensure that an error won't result in an exit. So it's a great help for cleanup and prepare steps that can fail if nothing is to be cleaned or prepared. The pattern is also reused in other tools, like [for instance systemd](https://github.com/rhdojun/simpleserver/blob/master/simpleserver.service#L8-L12).

Reply:
@rhdojun tyvm for the education!  :clinking_glasses: